scratchapi2/user.py

```diff
@@ -1,47 +1,53 @@
-#pylint: disable=trailing-whitespace

```

Comment:
Why would you ever disable the `trailing-whitespace` rule?

Reply:
If it's possible to disable only for docstrings, I will change pylintrc